reserver/util.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
```

Comment:
I suggest moving this function to `reserver_func.py` for better organization and clarity.